### PR TITLE
feat: declutter default layout

### PR DESCRIPTION
**Pull request by csandru-plenty**
## Why:

Closes: #ID

## Checklist before requesting a review

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: apps/web/__tests__/test/smoke/cartPage.cy.ts [hunk 1]
```diff
@@ -4,10 +4,12 @@ import { paths } from '../../../utils/paths';
...
 
 describe('Smoke: Cart Page', () => {
-
   beforeEach(() => {
     cy.setCookie('vsf-locale', 'en');
-    cy.setCookie('consent-cookie', '{"Essentials":{"Session":true,"Consent":true,"Session2":true},"External Media":{"Session":false,"Consent":false,"Session2":false},"Functional":{"Session":false,"Consent":false,"Session2":false},"Marketing":{"Session":false,"Consent":false,"Session2":false}}')
+    cy.setCookie(
+      'consent-cookie',
+      '{"Essentials":{"Session":true,"Consent":true,"Session2":true},"External Media":{"Session":false,"Consent":false,"Session2":false},"Functional":{"Session":false,"Consent":false,"Session2":false},"Marketing":{"Session":false,"Consent":false,"Session2":false}}',
+    );
```
Comment (abocsan-plenty) [inline apps/web/__tests__/test/smoke/cartPage.cy.ts:12]: Is this needed here ? do we do anything with these cookies ? Maybe it's best to consider removing them from here if we don't use them, because this structure is no longer valid anyway. Cookies have changed in `cookie.config.ts`

File: apps/web/components/CategoryFilters/Filter.vue [hunk 2]
```diff
@@ -118,10 +124,12 @@ function resetPriceFilter() {
 
 const updateFilter = () => {
   for (const filter of filters) {
-    models[filter.id.toString()] = Boolean(filter.selected) ?? false;
+    const filterId = typeof filter.id === 'string' ? filter.id : filter.id.toString();
```
Comment (abocsan-plenty) [inline apps/web/components/CategoryFilters/Filter.vue:127]: What was the specific scenario for which you needed to make this change ? 

It seems to me like if `filter.id` is `undefined` or `null` then this code will break.

If you are sure filter.id is always an integer or a string, then why did you need to check for string type ?

File: apps/web/components/CategoryPageContent/CategoryPageContent.vue [hunk 3]
```diff
@@ -11,7 +11,7 @@
         <div class="flex-1">
           <div class="flex justify-between items-center mb-6">
             <span class="font-bold font-headings md:text-lg">
-              {{ $t('numberOfProducts', { count: products?.length ?? 0, total: totalProducts }) }}
+              {{ i18n.t('numberOfProducts', { count: products?.length ?? 0, total: totalProducts }) }}
```
Comment (abocsan-plenty) [inline apps/web/components/CategoryPageContent/CategoryPageContent.vue]: Is there any specific reason we need to use `i18n` instead of `$t `? I understand we need to to that in composables to get rid of console error, but I would keep consistency in .vue files, and if we decide we want to switch from `$t` to `i18n` we would do it for all instances in the project.

File: apps/web/components/ui/ProductCard/ProductCard.vue [hunk 4]
```diff
@@ -4,17 +4,17 @@
...
+      <SfLink :tag="NuxtLink" rel="preload" :to="localePath(`${path}/${productSlug}`)" as="image">
         <NuxtImg
           :src="imageUrl"
           :alt="imageAlt"
           class="object-contain rounded-md aspect-square w-full h-full"
           data-testid="image-slot"
           width="190"
           height="190"
-          :loading="lazy && !priority ? 'lazy' : undefined"
+          :loading="lazy && !priority ? 'lazy' : ''"
```
Comment (abocsan-plenty) [inline apps/web/components/ui/ProductCard/ProductCard.vue]: I think **loading** parameter takes only 2 values: `lazy` and `eager`. Maybe the default should be set to `eager`.

**Review by abocsan-plenty — CHANGES_REQUESTED** [review]
Login page no longer looks the same on this branch as it did on main on **mobile**. There are 2 footers and the header title is different from a quick notice. We should make sure the ui doesn't change.

File: apps/web/components/CategoryFilters/Filter.vue (hunk 2 above)
Comment (csandru-plenty) [reply apps/web/components/CategoryFilters/Filter.vue:127]: Screenshot 2023-12-12 at 16 22 36
there's no addition to the code from main branch, just linting triggered that realigned the code

File: apps/web/components/CategoryPageContent/CategoryPageContent.vue (hunk 3 above)
Comment (csandru-plenty) [reply apps/web/components/CategoryPageContent/CategoryPageContent.vue]: typescript linting error
reference: https://github.com/jojomatik/nuxt-3-base/blob/4005fbc45b2304065080c22ffd70064bd82873ad/pages/login.vue

File: apps/web/components/ui/ProductCard/ProductCard.vue (hunk 4 above)
Comment (csandru-plenty) [reply apps/web/components/ui/ProductCard/ProductCard.vue]: linting error: source expects string, visible when building main vs current branch

File: apps/web/components/CategoryFilters/Filter.vue (hunk 2 above)
Comment (csandru-plenty) [reply apps/web/components/CategoryFilters/Filter.vue:127]: id can be either number or string, hence the reason for the check
error highlighted on main branch build

File: apps/web/__tests__/test/smoke/cartPage.cy.ts (hunk 1 above)
Comment (csandru-plenty) [reply apps/web/__tests__/test/smoke/cartPage.cy.ts:12]: Screenshot 2023-12-12 at 16 22 36
no code changes by comparison with main branch, just linting triggering code realignment

Comment (abocsan-plenty) [reply apps/web/__tests__/test/smoke/cartPage.cy.ts:12]: I understand, still this is bad code, we should check if we can remove it.

File: apps/web/components/CategoryFilters/Filter.vue (hunk 2 above)
Comment (abocsan-plenty) [reply apps/web/components/CategoryFilters/Filter.vue:127]: I have no errors on **main** branch `yarn build`, and still I don't understand why we need that check, it basically says if it is of type string, take it, otherwise transform it into a string, but `'anyString'.toString()` is still a string, so that check seems redundant to me. Let's maybe discuss this tomorrow.

File: apps/web/components/CategoryPageContent/CategoryPageContent.vue (hunk 3 above)
Comment (abocsan-plenty) [reply apps/web/components/CategoryPageContent/CategoryPageContent.vue]: I don't get any errors, not on console, not on `yarn build`, and not on `yarn lint` either, being on main branch. I also checked github lint actions and I don't see any errors regarding `$t` on this component. Let's maybe discuss it tomorrow.

File: apps/web/components/ui/ProductCard/ProductCard.vue (hunk 4 above)
Comment (abocsan-plenty) [reply apps/web/components/ui/ProductCard/ProductCard.vue]: I tested placing `eager` and it did not give me any linting error. I still think this is the best option, let's discuss tomorrow.

File: apps/web/__tests__/test/smoke/cartPage.cy.ts (hunk 1 above)
Comment (csandru-plenty) [reply apps/web/__tests__/test/smoke/cartPage.cy.ts:12]: there is a task created for that already

File: apps/web/components/ui/ProductCard/ProductCard.vue (hunk 4 above)
Comment (csandru-plenty) [reply apps/web/components/ui/ProductCard/ProductCard.vue]: above change has being pushed already, in my first reply I was just explaining the reason for the change

**Review by abocsan-plenty — APPROVED** [review]
